scripts/deblur

```diff
@@ -94,7 +94,12 @@ def trim(seqs_fp, output_fp, trim_len):
               show_default=True, help="Discard sequences with an abundance "
               "value smaller than min-size")
 def dereplicate(seqs_fp, output_fp, min_size):
-    """Step 2a: dereplicate FASTA sequences"""
+    """
+    Step 2a: dereplicate FASTA sequences.
```

Comment:
The reason for having the steps labeled here was so that they can appear in the command line help. This is something that was suggested in our deblur e-mail thread (march 12).